.ci/docker/common/install_libgomp.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Script used only in CD pipeline
+
+set -ex
+
+source /opt/rh/gcc-toolset-11/enable
```

Comment:
TODO: this is redundant given: https://github.com/pytorch/pytorch/pull/152361/files#diff-8904d8e261a20042944437665ad2f50f861b828aa8db2f961f4e76bc8b9c440bR49